qubes-rpc/qvm-copy-to-vm.kde

```diff
@@ -19,25 +19,33 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 #
+if type kdialog 2> /dev/null; then
+    kdlg=1
+    else kdlg=0
+fi
```

Comment:
I think it would be much more readable if the whole fallback is done here (no `kdialog` - call gnome version, maybe even `exec` it), instead of setting a variable to just check it once, a line later.